.github/workflows/ci.yml

```diff
@@ -47,6 +47,9 @@ jobs:
       - name: Sync
         run: yarn sync
 
+      - name: Sync check
+        run: git diff-index --quiet HEAD -- || (echo "Unstable sync result" && exit 1)
```

Comment:
What about showing. git diff in CI error?

Reply:
Actually I didn't know how to do that, googling 😅